hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestSCMNodeManagerMXBean.java

```diff
@@ -98,7 +98,7 @@ public void testNodeCount() throws Exception {
     Map<String, Map<String, Integer>> mbeanMap = convertNodeCountToMap(data);
     Map<String, Map<String, Integer>> nodeMap =
         scm.getScmNodeManager().getNodeCount();
-    assertTrue(nodeMap.equals(mbeanMap));
+    assertEquals(nodeMap, mbeanMap);
```

Comment:
Same a previous comment.

Reply:
I figured that `mbeanMap` is the one being verified here, since its a test for `MXBean`.